scrapers/AdultEmpire.yml

```
@@ -102,4 +108,31 @@ xPathScrapers:
                 - regex: ^
                   with: "https://www.adultdvdempire.com"
       URL: //meta[@name='og:url']/@content
-# Last Updated October 09, 2023
+  performerScraper:
+    common:
+      $infoPiece: //*[@id="profileModal"]/div/div/div[2]/div[1]/ul
+    performer:
+      Name: //*[@id="content"]/section/div/div[2]/h1/text()
+      Birthdate: $infoPiecel/li[contains(text(), 'Born:')]/text()
```

There's a typo here (an extra l in `$infoPiecel`) and we should try to remove the prefix:
```suggestion
      Birthdate:
        selector: $infoPiece/li[contains(text(), 'Born:')]/text()
        postProcess:
          - replace:
              - regex: Born:\s+(.*)
                with: $1
```